etc/scripts/json2csv.py

```diff
@@ -114,9 +114,6 @@ def collect_keys(mapping, key_group):
             if not path.endswith('/'):
                 path = path + '/'
 
-        # alway create a root directory
-        path = '/code' + path
```

Comment:
#JonoYang @pombredanne, as there is a user option, then the '/code' that I've removed from the test files, need to have an if-else option too, right?